packages/sdk-auth/src/auth.js

```diff
@@ -17,7 +16,7 @@ import * as constants from './constants'
 export default class SdkAuth {
   // Set flowtype annotations
   config: AuthOptions
-  fetcher: ConfigFetch
+  fetcher: typeof fetch
```

Comment:
We should be able to use the built-in type declarations for the fetch object.

Because of that, we can remove all our custom type declarations.